### PR TITLE
Modify Find item messages and test to match FoodRem instead of AddressBook

### PR DESCRIPTION
**Pull request by yixiann**
Rename instances of address book to FoodRem

Fixes: https://github.com/AY2223S1-CS2103T-W16-2/tp/issues/20

### CODE REVIEW

**Comment by Ferusel** [comment]
Hmm, can I ask what the other changes to the test files are? Since this PR is focused on mainly modifying the message to be displayed upon entering the Find command wrongly.

**Comment by Ferusel** [comment]
Also, may I ask how this fixes #20? What was the issue in #20 that this fixes?

**Comment by yixiann** [comment]
The find command was already working after changing person to item.
However, some of the test cases were still testing based on words of person instead of items. 

The test cases were originally passing as test to find is about matching keywords to one another and not unique to person so it was missed out.

I believe I have named the PR incorrectly. Have modified it.

File: src/main/java/seedu/foodrem/model/tag/UniqueTagList.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * A tag is considered unique by comparing using {@code Tag#equals(Tag)}. As such, adding and updating of
  * tags uses Tag#equals(Tag) for equality so as to ensure that the tag being added or updated is
  * unique in terms of identity in the UniqueTagList.
- *
+ * <p>
```
Comment (Ferusel) [inline src/main/java/seedu/foodrem/model/tag/UniqueTagList.java:20]: What is this <p>tag?

Comment (Ferusel) [inline src/main/java/seedu/foodrem/model/tag/UniqueTagList.java:20]: Hmm, is this "<p>" tag necessary? I'm not sure if this is used anywhere else in FoodREM

Comment (Ferusel) [reply src/main/java/seedu/foodrem/model/tag/UniqueTagList.java:20]: NOTE: Adds line break
image

**Review by Ferusel — APPROVED** [review]
LGTM!

**Pull request merged**